src/site/content/en/blog/reporting-api/index.md

```diff
@@ -0,0 +1,664 @@
+  Content-Security-Policy: script-src 'self'; object-src 'none'; report-to main-endpoint;
+```
+
+This post covers what this API can do, how it works, and how you can use it.
+
+Let's dive in!
+
+## Demo and code
+
+💥 [Demo site: report generation page](https://reporting-api-demo.glitch.me). This page
+uses the new Reporting API via the `Reporting-Endpoints` header. See the
+[code](https://glitch.com/edit/#!/reporting-api-demo).
+
+📬 [Demo site: reporting endpoint](https://reports-endpoint.glitch.me). This server
+receives and displays reports. See the
+[code](https://glitch.com/edit/#!/reports-endpoint).
+
+## Use cases and report types
+
+{% Banner 'caution', 'body' %}
+
+[Network Error Logging](https://w3c.github.io/network-error-logging/) isn't listed because it isn't supported in the new version of
+the API. Check the [migration guide](/reporting-api-migration/#network-error-logging) for details.
+
+{% endBanner %}
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>Report type</th>
+        <th>Example of a situation where a report would be generated</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP">CSP</a> violation (Level 3 only)</td>
+        <td>You've set a <code>Content-Security-Policy</code> on your site, but one of your pages is trying to load a script that's not allowed by your CSP.</td>
+      </tr>
+      <tr>
+        <td><a href="/why-coop-coep/#coop">COOP</a> violation</td>
+        <td>You've set a <code>Cross-Origin-Opener-Policy</code> on your site, but a cross-origin window is trying to interact directly with your document.</td>
+      </tr>
+      <tr>
+        <td><a href="/why-coop-coep/#coep">COEP</a> violation</td>
+        <td>You've set a <code>Cross-Origin-Embedder-Policy</code> on your site, but a page on your site includes a cross-origin iframe that has not opted into being loaded by cross-origin documents.</td>
+      </tr>
+      <tr>
+        <td>Document Policy violation</td>
+        <td>Your site has a document policy that prevents usage of <code>document.write</code>, but a script on your page tries to use <code>document.write</code>.</td>
+      </tr>
+      <tr>
+        <td>Deprecation warning</td>
+        <td>Your site is using an API that is deprecated, directly or via a top-level third-party script.
+        </td>
+      </tr>
+      <tr>
+        <td>Intervention</td>
+        <td>Your site is trying to do something that the browser decides not to honor, for security, performance or user experience reasons. Example in Chrome: your site uses <a href="https://developers.google.com/web/updates/2016/08/removing-document-write"><code>document.write</code> on slow networks</a> or uses <code>navigator.vibrate</code> in a cross-origin frame that the user hasn't interacted with yet.</td>
```

Comment:
This one I'm not sure if it applies to the site or the page.

Reply:
Thanks Eiji. I've included the suggestions. I'll check.